src/hooks/useAuthorityLinking/useAuthorityLinking.js

```diff
@@ -261,39 +274,85 @@ const useAuthorityLinking = ({ tenantId, marcType } = {}) => {
       return formValues;
     }
 
+    const searchParams = new URLSearchParams(search);
+    const isSharedRecord = searchParams.get('shared') === 'true';
     const linkedFields = formValues.records.filter(isFieldLinked);
     const payload = hydrateForLinkSuggestions(formValues, linkedFields);
-    const { fields: suggestedFields } = await fetchLinkSuggestions({
+    const isRecordWithCentralAndMemberSuggestions = marcType === MARC_TYPES.BIB && isMemberTenant && !isSharedRecord;

```

Comment:
We can call `useLinkSuggestions` hook twice with different `tenantId`, I think it'll look a bit cleaner
```js
const { fetchMemberLinkSuggestions, isLoading: isLoadingLinkSuggestions } = useLinkSuggestions({ marcType });
const { fetchCentralLinkSuggestions, isLoading: isLoadingLinkSuggestions } = useLinkSuggestions({ marcType, tenantId: centralTenantId });
...

const memberLinkSuggestionsPromise = fetchMemberLinkSuggestions(requestArgs);
const centralLinkSuggestionsPromise = isRecordWithCentralAndMemberSuggestions
  ? fetchCentralLinkSuggestions(requestArgs)
  : Promise.resolve({ fields: [] });
```